pkg/monitor/cluster/version.go

```diff
@@ -0,0 +1,35 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/util/restconfig"
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (mon *Monitor) emitClusterVersion(ctx context.Context) error {
+	restConfig, err := restconfig.RestConfig(mon.env, mon.oc)
+	if err != nil {
+		return err
+	}
+
+	configCli, err := configclient.NewForConfig(restConfig)
+	if err != nil {
+		return err
+	}
+
+	ver, err := configCli.ConfigV1().ClusterVersions().Get("version", metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	mon.emitGauge("cluster.version", 1, map[string]string{
+		"version": ver.Status.Desired.Version,
```

Comment:
do we just want version or image/pullspec as well?
https://github.com/Azure/ARO-RP/blob/master/pkg/util/version/version.go#L12

Reply:
another question
this is the desired version, so if a customer upgrades to something that does not exist or if an upgrade fails it will still show this?
wouldn't a better approach be to use the last completed history entry?

```
   Completion Time:    2020-03-26T01:10:03Z
    Image:              arosvc.azurecr.io/openshift-release-dev/ocp-release@sha256:64320fbf95d968fc6b9863581a92d373bc75f563a13ae1c727af37450579f61a
    Started Time:       2020-03-26T00:30:45Z
    State:              Completed
    Verified:           true
    Version:            4.3.5
```

Reply:
I agree with Angus, it seems like a better idea to get the last Completed history version, as that would be the authoritative current version.

The change would be simple. Would emitting an empty string be appropriate if there is no Completed event in the history?

Reply:
Returning both desired and actual state probably makes sense.  I am OK not reusing the code from #274 here at least for now.